polytracker/src/passes/polytracker_pass.cpp

```diff
@@ -231,7 +223,7 @@ bool PolytrackerPass::analyzeBlock(llvm::Function *func,
       &(*(func->getEntryBlock().getFirstInsertionPt()));
 
   llvm::IRBuilder<> IRB(insert_point);
-  llvm::Value *func_name = IRB.CreateGlobalStringPtr(func->getName());
+  IRB.CreateGlobalStringPtr(func->getName());
```

Comment:
Yeah, this smells like a NOOP to me.

Reply:
It's not strictly a NOOP because it inserts a string in the global scope of the module, at least according to the LLVM docs. Now, it's kind of hard for me to say if it's actually needed for anything anywhere. If you're sure it's not used anywhere, I can remove it.

Reply:
Since we don't have any reference to the string at this point I think it is safe to remove it.